esp-hal/src/uart.rs

```diff
@@ -241,6 +241,8 @@ pub struct Config {
     baudrate: u32,
     /// Determines how close to the desired baud rate value the driver should
     /// set the baud rate.
+    #[cfg_attr(not(feature = "unstable"), builder_lite(skip))]
```

Comment:
We should generate the skip attribute, too, I think. There is not much value in stating two different way that something is unstable.